src/patternfly/sass-utilities/mixins.scss

```diff
@@ -309,3 +309,12 @@
     }
   }
 }
+
+@mixin pf-v5-rtl($transform: null) {
+  @at-root :where(.ws-dir-rtl, .#{$pf-prefix}m-dir-rtl, [dir="rtl"]) & {
```

Comment:
Checked and looks to work well when applying `dir` attribute, `ws-dir-rtl` class, and the modifier class to either the `html` tag or the base component tag 👍🏼